source/georef/georef.h

```diff
@@ -151,6 +151,16 @@ struct Way :public nt::Nameable, nt::Header{
     }
     std::string get_label() const;
 
+#ifdef _DEBUG_DIJKSTRA_QUANTUM_
+    template <typename Stream, typename G>
+    void print(Stream& stream, const G& g){
+        for(auto& edge: this->edges){
+            stream << "LINESTRING(" << g[edge.first].coord.lon() << " " << g[edge.first].coord.lat()
```

Comment:
you can add `stream << std::setprecision(16) << "LINESTRING...` to have a correct precision.